es-ds-components/components/es-form-textarea.vue

```diff
@@ -0,0 +1,145 @@
+<script setup lang="ts">
+// Prevents attributes from being applied to first <div>
+// v-bind="$attr" is on the input instead
+defineOptions({
+    inheritAttrs: false,
+});
+
+const props = defineProps({
+    /**
+     * Required
+     */
+    required: {
+        type: Boolean,
+        default: false,
+    },
+    /**
+     * Disabled
+     */
+    disabled: {
+        type: Boolean,
+        default: false,
+    },
+    /**
+     * ID
+     * required
+     */
+    id: {
+        type: String,
+        required: true,
+    },
+    /**
+     * modelValue
+     */
+    modelValue: {
+        type: String,
+        required: true,
+        default: '',
```

Comment:
This custom code for `modelValue` and `update:modelValue` and `localValue` is not necessary.. All that's needed is to use `defineModel`, and set that value as `v-model` on the native `textarea`.

Reply:
To clarify, what I'm suggesting:
- remove `modelValue` prop
- Add `const model = defineModel<string>();`
- Remove `const emit...`
- Remove `const localValue...`
- Remove `watch(localValue...`
- Remove `watch(...`
- On the `textarea` component, replace `v-model="localValue"` with `v-model="model"`. Also apply `v-bind="$attrs"` as was stated on line 3